dvc/analytics.py

```diff
@@ -187,4 +187,4 @@ def _find_or_create_user_id():
             return user_id
 
     except LockError:
-        logger.debug(f"Failed to acquire '{lockfile}'")
+        logger.debug("Failed to acquire %r", lockfile)
```

Comment:
Here I have changed `'{lockfile}'` to `%r` (i.e. use the `repr`, as opposed to `%s`) which takes care of quotes and looks better if `lockfile` itself happens to contain single quotes. I've done this several times in this PR wherever the value appears to be a string.

Reply:
Oops, this caused tests to fail on Windows with good reason, as backslashes got escaped too, turning `'C:\\Users\\runneradmin\\AppData...'` into `'C:\\\\Users\\\\runneradmin\\\\AppData...`. So I guess I'll revert all of these to just `'%s'`. Any objections?